src/Doctrine/ORM/GiftCardRepository.php

```diff
@@ -12,11 +12,14 @@
 use Sylius\Component\Core\Model\CustomerInterface;
 use Sylius\Component\Core\Model\OrderItemUnitInterface;
 
-final class GiftCardRepository extends EntityRepository implements GiftCardRepositoryInterface
+/** not final */ class GiftCardRepository extends EntityRepository implements GiftCardRepositoryInterface
```

Comment:
This was an error in the first place. Just remove this comment. We won't let it happen again :D